keyboards/xd75/keymaps/colinta/.gitignore

```diff
@@ -0,0 +1 @@
+colinta.h
```

Comment:
Would you mind sticking this in your `.git/info/exclude` file instead? That'll help keep things cleaner in the repo :) [more info here!](https://help.github.com/articles/ignoring-files/#explicit-repository-excludes)

Reply:
Oh yeah can do - I left it in case someone wanted to take advantage of this feature, but i'll just update the readme.